runtime/helpers/dateTimeHelpers.js

```diff
@@ -0,0 +1,138 @@
+'use strict';
+
+
+module.exports = {
+
+/**
+		* Converting String date into the Date format
+		*
+		* @param _date : String date that user passes in
+		* @param _format : "dd/MM/yyyy", "mm/dd/yyyy", "mm-dd-yyyy"
+		* @param _delimiter
+		* @returns {Date}
+		*
+		* Example use
+		*
+		* stringToDate("17/9/2014","dd/MM/yyyy","/");
+		* stringToDate("9/17/2014","mm/dd/yyyy","/")
+		* stringToDate("9-17-2014","mm-dd-yyyy","-")
+		*/
+        stringToDate: function (_date, _format, _delimiter) {
+            let formatLowerCase = _format.toLowerCase();
```

Comment:
I think in the linter rules it is not allowed to use _ before an variable. Maybe we have to include the linter roles to integration tests

Reply:
Corrected with the latest commit and push